DataRepo/templates/upload.html

```diff
@@ -1,50 +1,109 @@
-            </ul>
+        <li>Download your Animal and Sample Table in Excel format.</li>
+        <li>Submit the table and any AccuCor files on the <a href="{% url 'validate' %}">TraceBase validation page</a></li>
+        <li>Fix any errors reported, re-download the table and validate the new Excel file.<br/>
+            Note that in general, you should <strong>NOT</strong> edit your AccuCor
+            files. It&#39;s simplest and often best to resolve issues by
+            editing your animal and sample table file.
         </li>
-        <li>
-            <h5>Optional: <a href="{% url 'validate' %}" target="_blank">Validate</a> your data</h5>
+    </ol>
+    <h5 id="submit-your-data-on-google-drive">Submit your data on Google Drive</h5>
+    <ol>
+        <li>Navigate to the <a target="_blank"
+                href="https://drive.google.com/drive/folders/1cBy3eezfr_0vmaz8RGodga6A8n3hkm1m?usp=sharing">TraceBase
+                Study Submission folder</a> on Google Drive.  </li>
+        <li>Create a folder with your name (if it doesn&#39;t already exist).</li>
+        <li>Create a new folder for this study.</li>
+        <li>Add the following files to the study folder:
             <ul>
-                <li><strong>DO NOT</strong> edit your accucor files.  Resolve issues by editing your animal and sample table file.</li>
-                <li>In order to expedite the loading process, you can check your data submission to resolve simple discrepancies by running your files through our automated validation script.</li>
-                <li>Note, if you saved your animal and sample table in a google spreadsheet, you must download it as an excel file to supply it to the data validation form.</li>
+                <li>TraceBase Animal and Sample spread sheet</li>
+                <li>Any AccuCor files that should be included in the study</li>
             </ul>
         </li>
-        <li>
-            <h5>Upload your data using the <a href="{{ data_submission_url }}" target="_blank">submission form</a></h5>
-            Submitting this form will do a few things:
-            <ol>
-                <li>Submitting the form will record your entries in a google spreadsheet and store your file uploads in google drive</li>
-                <li>The submission will notify the Tracebase curators responsible for manually loading your submission into the database.</li>
-                <li>You will receive an email confirming your data submission.</li>
-            </ol>
-            You can check what you submitted by going to the google spreadsheet.  If you have edit permission on the spreadsheet, you can alter any values that may need editing.
+        <li>Submit the <a href="https://forms.gle/vEfJEfhPbCbkybpE7">TraceBase Study Submission
+            form</a> <a class="badge bg-secondary" data-bs-toggle="collapse" data-bs-target="#study-submission-details">more info</a>
```

Comment:
NB. This is the first reference to the submission form.  useing `the` implies it's a reference to a previously explained form (though it doesn't appear in the [unexpanded?] instructions above).
```suggestion
        <li>Fill out a <a href="https://forms.gle/vEfJEfhPbCbkybpE7">TraceBase Study Submission
            form</a> <a class="badge bg-secondary" data-bs-toggle="collapse" data-bs-target="#study-submission-details">more info</a>
```